app/code/community/HE/TwoFactorAuth/Model/Observer.php

```diff
@@ -171,7 +171,7 @@ public function googleClearSecretCheck(Varien_Event_Observer $observer)
 
         if ($block->getType() == 'adminhtml/permissions_user_edit_form') {
 
-            // check that google is set for twofactor authentication            
+            // check that google is set for twofactor authentication
```

Comment:
Let's not change this indentation for no reason. Force push this out, I don't want it in the `git blame`.